Purchases.xcodeproj/project.pbxproj

```diff
@@ -1480,7 +1480,7 @@
 				352629FA1F7C4B9100C04F2C /* Frameworks */,
 				352629FB1F7C4B9100C04F2C /* Headers */,
 				352629FC1F7C4B9100C04F2C /* Resources */,
-				B385A1532674324500C6F132 /* ShellScript */,
+				B385A1532674324500C6F132 /* Swiftlint */,
```

Comment:
added a name for the script so it's easier to understand which one it is if it fails